packages/default-field-editors/tsconfig.json

```
@@ -4,7 +4,6 @@
   "compilerOptions": {
     "module": "esnext",
     "moduleResolution": "bundler",
```

```suggestion
    "module": "Node16",
    "moduleResolution": "Node16",
```

That way you get a typescript error when you forget about the `.js` extension